mods/ts/sequences/infantry.yaml

```diff
@@ -526,12 +526,37 @@ vislrg:
 		ShadowStart: 90
 		Tick: 80
 		Offset: 0, 0, 16
-	muzzle: vislgatk # TODO: unused
+	attack:
+		Combine:
+			vislgatk:
```

Comment:
This looks mostly redundant... why can't you merge the first 7 into a single `Start: 5` `Length: 35`?